lib/ddtrace/contrib/aws/patcher.rb

```diff
@@ -0,0 +1,56 @@
+module Datadog
+  module Contrib
+    module Aws
+      SERVICE = 'aws'.freeze
+      AGENT = 'aws-sdk-ruby'.freeze
+      RESOURCE = 'aws.command'.freeze
+
+      # Responsible for hooking the instrumentation into aws-sdk
+      module Patcher
+        @patched = false
+
+        class << self
+          def patch
```

Comment:
@p-lambert cool! the code is fine, now we're only missing the documentation and the integration is ready. Feel free to ping me again to review the docs. Thank you!